common/eq_packet_structs.h

```diff
@@ -1619,6 +1619,32 @@ struct MoveItem_Struct
 /*0012*/
 };
 
+// New for RoF2 - Size: 12
+struct InventorySlot_Struct
+{
+/*000*/	int16	Type;		// Worn and Normal inventory = 0, Bank = 1, Shared Bank = 2, Delete Item = -1
+/*002*/	int16	Unknown02;
+/*004*/	int16	Slot;
+/*006*/	int16	SubIndex;
+/*008*/	int16	AugIndex;	// Guessing - Seen 0xffff
+/*010*/	int16	Unknown01;	// Normally 0 - Seen 13262 when deleting an item, but didn't match item ID
```

Comment:
`snake_case` please.

Reply:
This exists already in `rof_structs.h`, I just copied it. Would it be more correct to adjust this in `rof_structs.h` and reference it from there?

Reply:
Sure, whatever works.

Reply:
I do not want to own a change that touches as much code as would be affected by this.

Reply:
Fair